### DISCUSSION TITLE
Welcome to RIA2_rekognition_python Discussions!

### DISCUSSION THREAD

**Discussion by SuperQuentin** [CPNV-ES/RIA2_rekognition_python#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by QuentinNev** [comment]
Coucou

**Comment by MathieuBurnat** [comment]
Coucou @SuperQuentin, tu es super ❤️
